### PR TITLE
*: update resource control about background example

### PR DESCRIPTION
**Pull request by nolouch**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by glorv — COMMENTED** [review]
LGTM

File: system-variables.md [hunk 1]
```diff
@@ -4782,6 +4782,14 @@ Query OK, 0 rows affected, 1 warning (0.00 sec)
 - 这个变量用于设置 window 算子的并行度。
 - 默认值 `-1` 表示使用 `tidb_executor_concurrency` 的值。
 
+### `tidb_request_source_type` <span class="version-mark">从 v7.4.0 版本开始引入</span>
```
Comment (Oreoxmt) [inline system-variables.md:4785]: @nolouch Please add `tidb_request_source_type` to release-7.4.0.md:
https://github.com/pingcap/docs-cn/blob/a9062ca0c6401a9674f074157c055505ce905e13/releases/release-7.4.0.md?plain=1#L289-L304

File: system-variables.md [hunk 2]
```diff
@@ -4782,6 +4782,14 @@ Query OK, 0 rows affected, 1 warning (0.00 sec)
...
+
+- 作用域：SESSION
+- 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
+- 类型：字符串
+- 默认值：`""`
```
Comment (Oreoxmt) [inline system-variables.md:4790]:
```suggestion
- 默认值：`""`
- 可选值：
```

File: tidb-resource-control.md [hunk 3]
```diff
@@ -368,33 +366,53 @@ Runaway Query 是指执行时间或消耗资源超出预期的查询。下面使
 - `br`：使用 [BR](/br/backup-and-restore-overview.md) 执行数据备份和恢复。目前不支持 PITR。
 - `ddl`：对于 Reorg DDL，控制批量数据回写阶段的资源使用。
 - `stats`：对应手动执行或系统自动触发的[收集统计信息](/statistics.md#统计信息的收集)任务。
+- `background`: 预留的一个任务类型，可使用 [tidb_request_source_type](/system-variables.md#tidb_request_source_type-从-v740-版本开始引入) 系统变量指定当前会话的任务类型为 `background`.
```
Comment (Oreoxmt) [inline tidb-resource-control.md:369]:
```suggestion
- `background`：预留的一个任务类型，可使用 [`tidb_request_source_type`](/system-variables.md#tidb_request_source_type-从-v740-版本开始引入) 系统变量指定当前会话的任务类型为 `background`.
```

File: tidb-resource-control.md [hunk 4]
```diff
@@ -368,33 +366,53 @@ Runaway Query 是指执行时间或消耗资源超出预期的查询。下面使
...
+
+默认情况下，被标记为后台任务的任务类型为`空`，此时后台任务的管理功能处于关闭状态。如需开启后台任务管理功能，你需要手动修改 `default` 资源组的后台任务类型以开启后台任务管理。后台任务类型被识别匹配后，资源控制会自动控制。具体行为为当系统资源紧张，后台任务会自动降为最低优先级，保证前台任务的执行。
```
Comment (Oreoxmt) [inline tidb-resource-control.md:371]:
```suggestion
默认情况下，被标记为后台任务的任务类型为 `""`，此时后台任务的管理功能处于关闭状态。如需开启后台任务管理功能，你需要手动修改 `default` 资源组的后台任务类型以开启后台任务管理。后台任务类型被识别匹配后，资源管控会自动进行，即当系统资源紧张时，后台任务会自动降为最低优先级，保证前台任务的执行。
```

File: system-variables.md [hunk 5]
```diff
@@ -4782,6 +4782,14 @@ Query OK, 0 rows affected, 1 warning (0.00 sec)
...
+- 显示指定当前会话的任务类型，用于资源控制能识别并进行控制。如 `set @@tidb_request_source_type = "background"`。
```
Comment (Oreoxmt) [inline system-variables.md:4791]:
```suggestion
- 显示指定当前会话的任务类型，用于[资源管控](/tidb-resource-control.md)识别并控制。如 `SET @@tidb_request_source_type = "background"`。
```

File: tidb-resource-control.md [hunk 6]
```diff
@@ -368,33 +366,53 @@ Runaway Query 是指执行时间或消耗资源超出预期的查询。下面使
...
 
-默认情况下，被标记为后台任务的任务类型为空，此时后台任务的管理功能处于关闭状态，其行为与 TiDB v7.4.0 之前版本保持一致。你需要手动修改 `default` 资源组的后台任务类型以开启后台任务管理。
+> **注意：**
+>
+> 目前所有资源组的后台任务默认都会绑定到默认资源组 `default` 下面进行管控，可以通过 `default` 全局管控后台任务类型，还不支持将后台任务绑定到其他资源组。
```
Comment (Oreoxmt) [inline tidb-resource-control.md:375]:
```suggestion
> 目前，所有资源组的后台任务默认都会绑定到默认资源组 `default` 下进行管控，你可以通过 `default` 全局管控后台任务类型。暂不支持将后台任务绑定到其他资源组。
```

File: tidb-resource-control.md [hunk 7]
```diff
@@ -368,33 +366,53 @@ Runaway Query 是指执行时间或消耗资源超出预期的查询。下面使
...
 
     ```sql
-    ALTER RESOURCE GROUP rg1 BACKGROUND=(TASK_TYPES="");
+    mysql> select * from information_schema.resource_groups where name="default";
+    +---------+------------+----------+-----------+-------------+---------------------+
+    | NAME    | RU_PER_SEC | PRIORITY | BURSTABLE | QUERY_LIMIT | BACKGROUND          |
+    +---------+------------+----------+-----------+-------------+---------------------+
+    | default | UNLIMITED  | MEDIUM   | YES       | NULL        | TASK_TYPES='br,ddl' |
+    +---------+------------+----------+-----------+-------------+---------------------+
+    ```
```
Comment (Oreoxmt) [inline tidb-resource-control.md:406]:
````suggestion
    SELECT * FROM information_schema.resource_groups WHERE NAME="default";
    ```

输出结果如下：

    ```
    +---------+------------+----------+-----------+-------------+---------------------+
    | NAME    | RU_PER_SEC | PRIORITY | BURSTABLE | QUERY_LIMIT | BACKGROUND          |
    +---------+------------+----------+-----------+-------------+---------------------+
    | default | UNLIMITED  | MEDIUM   | YES       | NULL        | TASK_TYPES='br,ddl' |
    +---------+------------+----------+-----------+-------------+---------------------+
    ```
````

File: tidb-resource-control.md [hunk 8]
```diff
@@ -368,33 +366,53 @@ Runaway Query 是指执行时间或消耗资源超出预期的查询。下面使
...
+
+5. 某些情况下，用户也希望显示将会话里面的任务标记为后台类型。使用 `tidb_request_source_type`，可以显示指定任务类型，如：
+
+    ``` sql
+    set @@tidb_request_source_type="background";
+    /* then we can add task type */
+    alter resource group default background=(task_types="background");
+    /* then run the command in this session */
+    load data infile "s3://resource-control/Lightning/test.customer.aaaa.csv"
     ```
```
Comment (Oreoxmt) [inline tidb-resource-control.md:416]:
```suggestion
5. 如果希望显示将当前会话里的任务标记为后台类型，你可以使用 `tidb_request_source_type` 显示指定任务类型，如：

    ``` sql
    SET @@tidb_request_source_type="background";
    /* 添加 background 任务类型 */
    ALTER RESOURCE GROUP `default` BACKGROUND=(TASK_TYPES="background");
    /* 在当前会话中执行 LOAD DATA */
    LOAD DATA INFILE "s3://resource-control/Lightning/test.customer.aaaa.csv"
    ```
```

File: system-variables.md (hunk 1 above)
Comment (hfxsd) [reply system-variables.md:4785]: Done

File: system-variables.md [hunk 9]
```diff
@@ -4086,6 +4086,15 @@ EXPLAIN FORMAT='brief' SELECT COUNT(1) FROM t WHERE a = 1 AND b IS NOT NULL;
 - 这个变量用于控制 TiDB 的 Follower Read 功能的行为。
 - 关于使用方式与实现原理，见 [Follower Read](/follower-read.md)。
 
+### `tidb_request_source_type` <span class="version-mark">从 v7.4.0 版本开始引入</span>
+
+- 作用域：SESSION
+- 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
+- 类型：字符串
+- 默认值：`""`
+- 可选值：
```
Comment (hfxsd) [inline system-variables.md:4095]: 需要加上可选值

Comment (nolouch) [inline system-variables.md:4095]:
```suggestion
- 可选值："ddl", "stats", "br", "lightning"
```

File: system-variables.md [hunk 10]
```diff
@@ -4086,6 +4086,15 @@ EXPLAIN FORMAT='brief' SELECT COUNT(1) FROM t WHERE a = 1 AND b IS NOT NULL;
 - 这个变量用于控制 TiDB 的 Follower Read 功能的行为。
 - 关于使用方式与实现原理，见 [Follower Read](/follower-read.md)。
 
+### `tidb_request_source_type` <span class="version-mark">从 v7.4.0 版本开始引入</span>
+
+- 作用域：SESSION
+- 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
+- 类型：字符串
+- 默认值：`""`
+- 可选值："ddl", "stats", "br", "lightning", "background"
```
Comment (Oreoxmt) [inline system-variables.md:4095]:
```suggestion
- 可选值：`"ddl"`、`"stats"`、`"br"`、`"lightning"`、`"background"`
```

**Review by Oreoxmt — APPROVED** [review]

**Review by hfxsd — APPROVED** [review]

**Pull request merged**